runtime/frequency/src/lib.rs

```diff
@@ -498,11 +496,16 @@ impl pallet_balances::Config for Runtime {
 	type WeightInfo = weights::pallet_balances::SubstrateWeight<Runtime>;
 	type MaxReserves = BalancesMaxReserves;
 	type ReserveIdentifier = [u8; 8];
+	type MaxHolds = ConstU32<0>;
+	type MaxFreezes = ConstU32<0>;
```

Comment:
These zeros feel wrong. Double checking that they are correct.

Reply:
As I recall from the release notes, that was the recommended value until your pallet starts using the values.

Reply:
@shannonwells I think we are not using this until your later work to update for `Fungible` right?

Reply:
10-4. Then @mattheworris we are in the clear. Everything uses reserves currently